### PR TITLE
(686) Add production helm charts & rename the service to match

### PR DESCRIPTION
**Pull request by tahb**
# Context

https://trello.com/c/EFt2j3Lz

# Changes in this PR

[We've confirmed that the service name should be "Short-Term Accommodation (CAS-2)" in this Slack thread](https://mojdt.slack.com/archives/C04T6SC31HA/p1704812197680799) which specifically [references this as the result of a session in July 2023](https://mojdt.slack.com/archives/C04T6SC31HA/p1686922608645579).

- Add production helm charts, checking that maintenance mode is set to true
- Rename the service name to match the new domain name, this updates the header and page titles

Much of the content uses "CAS-2" in what appears to be a short-hand reference, this remains as it was and reduces the size of this change quite a bit. We attempt to rename only the fuller/longer description of the service and only from the users point of view. We keep all internal references of `community-accommodation-tier-2` to match the repository, the readme and any internal references. This not only makes this change easier to make now but it means we can rename it again in future without having to update everything again.

### Before

Screenshot 2024-01-10 at 10-20-46 Home - CAS-2

### After

Screenshot 2024-01-10 at 10-21-44 Home - Short-Term Accommodation (CAS-2)

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: server/views/applications/before-you-start.njk [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1>Apply for a CAS-2 placement</h1>
-      <p>Community Accommodation Service Tier 2 (CAS-2) provides housing for people who do not have a suitable address for the term of their licence or Bail Order.</p>
+      <p>Short-Term Accommodation (CAS-2/Tier-2) provides housing for people who do not have a suitable address for the term of their licence or Bail Order.</p>
```
Comment (tahb) [inline server/views/applications/before-you-start.njk:12]: I had to make a decision here given "Tier-2" already existed. I'm not convinced about adding it on as another part of the brackets but it felt like a safer stance to take?

We could of course remove it in favour of "Short-Term Accommodation (CAS-2)" or something else though as this page appears to be a start page it felt like a good place to relate to as many people as possible.

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+import { initialiseName, removeBlankSummaryListItems } from './utils'
 
 const production = process.env.NODE_ENV === 'production'
 
 export default function nunjucksSetup(app: express.Express, path: pathModule.PlatformPath): void {
   app.set('view engine', 'njk')
 
   app.locals.asset_path = '/assets/'
-  app.locals.applicationName = 'CAS-2'
+  app.locals.applicationName = 'Short-Term Accommodation (CAS 2)'
```
Comment (tahb) [inline server/utils/nunjucksSetup.ts:35]: With `short-term-accommodation-cas-2` as the domain name we need to choose how we present the service name that appears in the title. In particular what do we do about the "CAS-2" part? 

1. Short-Term Accommodation (CAS 2)
2. Short-Term Accommodation - CAS 2
3. Short-Term Accommodation CAS 2
4. Other?

"CAS-2" appears throughout as a secondary/short-hand name so it seemed reasonable to put it in brackets here as a way to connect the long and short hand name?

File: server/views/applications/before-you-start.njk (hunk 1 above)
Comment (tahb) [reply server/views/applications/before-you-start.njk:12]: @chaneldiepmoj what do you think?

File: server/utils/nunjucksSetup.ts (hunk 2 above)
Comment (tahb) [reply server/utils/nunjucksSetup.ts:35]: @chaneldiepmoj what do you think?

File: server/views/applications/before-you-start.njk (hunk 1 above)
Comment (tahb) [reply server/views/applications/before-you-start.njk:12]: [Conversation continued in the Slack thread](https://mojdt.slack.com/archives/C04T6SC31HA/p1704821481047049?thread_ts=1704812197.680799&cid=C04T6SC31HA)

File: server/utils/nunjucksSetup.ts (hunk 2 above)
Comment (tahb) [reply server/utils/nunjucksSetup.ts:35]: [Conversation continued in the Slack thread](https://mojdt.slack.com/archives/C04T6SC31HA/p1704821481047049?thread_ts=1704812197.680799&cid=C04T6SC31HA)

File: helm_deploy/values-prod.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  namespace_secrets:
+    sqs-hmpps-audit-secret:
+      AUDIT_SQS_QUEUE_URL: 'sqs_queue_url'
+      AUDIT_SQS_QUEUE_NAME: 'sqs_queue_name'
+
+  allowlist: null
+
+generic-prometheus-alerts:
+  alertSeverity: digital-prison-service-dev
```
Comment (Harriethw) [inline helm_deploy/values-prod.yaml:29]: I don't know if we use these prometheus alerts, but should this be `digital-prison-service` ?

Comment (tahb) [reply helm_deploy/values-prod.yaml]: Good shout, this was actually removed for prod by cas3 after a convo with the platform team. I've removed it and edited the commit to include the relevant links.

**Review by Harriethw — APPROVED** [review]
🚀

**Comment by tahb** [comment]
The new certificate has now been created here https://github.com/ministryofjustice/cloud-platform-environments/pull/18579

**Pull request merged**